File: CommonLanguageExensions/BoolExtensions.cs
namespace CommonLanguageExtensions
{
    /// <summary>
    /// Language extensions for type bool
    /// </summary>
    public static class BoolExtensions
    {
        /// <summary>
        /// Convert bool to English text
        /// </summary>
        /// <param name="value"></param>
        /// <returns></returns>
        public static string ToYesNoString(this bool value) => value ? "Yes" : "No";
    }
}

File: CommonLanguageExtensions_vb/BoolExtensions.vb
Imports System.Runtime.CompilerServices

Namespace CommonLanguageExtensions
    ''' <summary>
    ''' Boolean code samples
    ''' </summary>
    Public Module BoolExtensions
        ''' <summary>
        ''' Convert bool to English text
        ''' </summary>
        ''' <param name="value"></param>
        ''' <returns>Yes or No as a string</returns>
        <Extension>
        Public Function ToYesNoString(value As Boolean) As String
            Return If(value, "Yes", "No")
        End Function
    End Module
End Namespace
